src/main/java/seedu/multitasky/model/StorageUserPrefs.java

```diff
@@ -0,0 +1,43 @@
+package seedu.multitasky.model;
+
+//@@author A0132788U
+/**
+ * Represents the default values in UserPrefs object that storage has access to.
+ */
+public interface StorageUserPrefs {
+
+    public String getEntryBookFilePath();
+
+    public void setEntryBookFilePath(String entryBookFilePath);
+
+    /**
+     * The getter and setter for the snapshot file path
+     */
+    public static String getEntryBookSnapshotPath() {
+        return UserPrefs.getEntryBookSnapshotPath();
```

Comment:
Static methods cannot be overriden,I already tried it. You cannot declare static methods in an interface without defining them or changing the method signature to non-static (which is difficult to do as the Storage class needs to keep track of the snapshots for each instance)
It was either being dependent on UserPrefs or leaving the implementations blank, and Codacy was giving an error if i left the fields blank.